tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-card.html

```diff
@@ -0,0 +1,288 @@
+    process-data="[[_processData]]"
+  >
+    <div class="buttons">
+      <paper-icon-button
+        selected$="[[_expanded]]"
+        icon="fullscreen"
+        on-tap="_toggleExpanded"
+      ></paper-icon-button>
+      <paper-icon-button
+        selected$="[[_logScaleActive]]"
+        icon="line-weight"
+        on-tap="_toggleLogScale"
+        title="Toggle y-axis log scale"
+      ></paper-icon-button>
+      <paper-icon-button
+        icon="settings-overscan"
+        on-tap="_resetDomain"
+        title="Fit domain to data"
+      ></paper-icon-button>
+      <span style="flex-grow: 1"></span>
+      <template is="dom-if" if="[[showDownloadLinks]]">
+        <div class="download-links">
+          <paper-dropdown-menu
+            no-label-float="true"
+            label="run to download"
+            selected-item-label="{{_runToDownload}}"
+          >
+            <paper-menu class="dropdown-content" slot="dropdown-content">
+              <template is="dom-repeat" items="[[runs]]">
+                <paper-item no-label-float=true>[[item]]</paper-item>
+              </template>
+            </paper-menu>
+          </paper-dropdown-menu>
+          <a
+            download="run_[[_runToDownload]]-tag-[[tag]].csv"
+            href="[[_csvUrl(_runToDownload)]]"
+          >CSV</a> <a
+            download="run_[[_runToDownload]]-tag-[[tag]].json"
+            href="[[_jsonUrl(_runToDownload)]]"
+          >JSON</a>
+          </div>
+        </div>
+      </template>
+    </div>
+  </tf-scalar-chart>
+  <style>
+    :host {
+      height: 235px;
+      width: 330px;
+      margin: 5px;
+      display: block;
+    }
+
+    :host[_expanded] {
+      height: 400px;
+      width: 100%;
+    }
+
+    paper-icon-button {
+      color: #2196F3;
+      border-radius: 100%;
+      width: 32px;
+      height: 32px;
+      padding: 4px;
+    }
+
+    paper-icon-button[selected] {
+      background: var(--tb-ui-light-accent);
+    }
+
+    .download-links {
+      display: flex;
+      height: 32px;
+    }
+
+    .download-links a {
+      font-size: 10px;
+      align-self: center;
+      margin: 2px;
+    }
+
+    .download-links paper-dropdown-menu {
+      width: 100px;
+      --paper-input-container-label: {
+        font-size: 10px;
+      }
+      --paper-input-container-input: {
+        font-size: 10px;
+      }
+    }
+  </style>
+</template>
+<script>
+  import * as ChartHelpers from '../vz-line-chart/vz-chart-helpers.js';
+  import {getRouter} from '../tf-backend/router.js';
+
+  /** 
+   * Allows:
+   * - "step" - Linear scale using the "step" property of the datum.
+   * - "wall_time" - Temporal scale using the "wall_time" property of the
+   * datum.
+   * - "relative" - Temporal scale using the "relative" property of the
+   * datum if it is present or calculating from "wall_time" if it isn't.
+   * @enum {string}
+   * */
+  
+  const X_TYPE = {
+    STEP: 'step',
+    RELATIVE: 'relative',
+    WALL: 'wall',
```

Comment:
`wall_time`? (Not that the values of `X_TYPE` are ever used.)

Reply:
Done.